hadoop-hdfs-project/hadoop-hdfs/src/main/resources/hdfs-default.xml

```diff
@@ -4955,6 +4955,19 @@
   </description>
 </property>
 
+<property>
+  <name>dfs.journalnode.edit-cache-size.fraction</name>
+  <value>0.5f</value>
+  <description>
+    This ratio refers to the proportion of the maximum memory of the JVM.
+    Used to calculate the size of the edits cache that is kept in the JournalNode's memory.
```

Comment:
we should explicitly mention that this is an alternative to the `.bytes` config and either copy some of the guidance there (about when it will be enabled, txn size, etc.) or reference it like "see the documentation for `dfs.journalnode.edit-cache-size.bytes` to learn more"